tests/features/stable20.feature

```diff
@@ -1,54 +0,0 @@
-Feature: CLI updater - stable20 base
-
-  Scenario: Update is available - 20.0.0 beta 1 to 20.0.0 RC 1
-    Given the current installed version is 20.0.0beta1
-    And there is an update to prerelease version "20.0.0RC1" available
-    And the version number is decreased in the config.php to enforce upgrade
-    When the CLI updater is run successfully
-    And the output should contain "Update successful"
-    Then the installed version should be 20.0
-    And maintenance mode should be off
-    And upgrade is not required
-
-  Scenario: Update is available but unexpected folder found - 20.0.7 to 20.0.8
```

Comment:
unexpected folder here.

All these tests should be kept IMO since they test features of the updater.